docs/guides/Storyboards.md

```diff
@@ -1,23 +0,0 @@
-# Setting up Mapbox Navigation in a storyboard
```

Comment:
Looks like this PR deletes the storyboard guide, yet it remains in the table of contents under “Examples”. We should keep the storyboard guide as a guide – that is, under docs/guides/ and the “Guides” section, separate from the examples.